eZ/Publish/Core/FieldType/Tests/FieldTypeTest.php

```diff
@@ -974,5 +979,20 @@ protected function doValidate($fieldDefinitionData, $value)
         return $validationErrors;
     }
 
+    /**
+     * @dataProvider provideInputForValuesEqual
+     */
+    public function testValuesEqual($inputValue1, SPIValue $inputValue2): void
```

Comment:
This wasn't resolved: https://github.com/ezsystems/ezpublish-kernel/pull/3060#discussion_r514418037

Reply:
Changed